app/assets/v2/js/grants/index.js

```diff
@@ -23,9 +23,9 @@ $(document).ready(() => {
   waitforWeb3(() => {
     let _network = $('#grant-network').html();
 
-    $('#sub_tx_link').attr('href', etherscan_tx_url($('#sub_tx_link').attr('href'), _network));
-    $('#tx_link').attr('href', etherscan_tx_url($('#tx_link').attr('href'), _network));
-    $('#cancel_tx_link').attr('href', etherscan_tx_url($('#cancel_tx_link').attr('href'), _network));
+    $('.sub_tx_link').attr('href', etherscan_tx_url($('.sub_tx_link').attr('href'), _network));
```

Comment:
this sets the HREF of each . sub_tx_link to the first `$('.sub_tx_link').attr('href')` that it finds.

if you test this, it makes all the links on the page the same.

 you need to wrap this into a `$('.sub_tx_link').each()` to workaround it